### DISCUSSION TITLE
Deploy Masterclass PWAs com Blazor WASM

### DISCUSSION THREAD

**Discussion by lucasmiko** [balta-io/forum#495] [Tira Dúvidas] [answered]
Após o deploy do projeot no github, ele não está funcionando. Localmente está ok, como na aula, mas no github não.

Base local:

![image](https://github.com/balta-io/forum/assets/63825231/d9e9232f-e7c5-4ab2-91c5-2774787b2f8a)

Deploy no github:

![image](https://github.com/balta-io/forum/assets/63825231/43091d84-ab44-437b-9789-dca4ea7735cf)

Link do projeto: https://lucasmiko.github.io/calculadora-imc-blazor-wasm-pwa/
Link do repositório: https://github.com/lucasmiko/calculadora-imc-blazor-wasm-pwa

Alguém enfrentou o mesmo problema e tem alguma sugestão? Obrigado desde já pela ajuda.

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @lucasmiko, como vai?

Acredito que no `<base href` no seu `index.html` ficou faltando um `/` no fim.

O seu base atual:
```
<base href="https://lucasmiko.github.io/calculadora-imc-blazor-wasm-pwa" />
```

Faz um teste mudando para isto:
```
<base href="https://lucasmiko.github.io/calculadora-imc-blazor-wasm-pwa/" />
```

**Reply by lucasmiko** [reply]
Era isso mesmo, 🤡 hehe.

Obrigado Balta.
